health.go

```diff
@@ -0,0 +1,51 @@
+// Copyright (c) Mainflux
+// SPDX-License-Identifier: Apache-2.0
+
+package mainflux
+
+import (
+	"encoding/json"
+	"net/http"
+)
+
+const (
+	version         string = "0.12.1"
+	contentType            = "Content-Type"
+	contentTypeJSON string = "application/json"
+	svcStatus       string = "pass"
+)
+
+// HealthInfo contains version endpoint response.
+type HealthInfo struct {
+	// Status contains service status.
+	Status string `json:"status"`
+
+	// Version contains current service version.
+	Version string `json:"version"`
+
+	// Description contains service description.
+	Description string `json:"description"`
+}
+
+// Health exposes an HTTP handler for retrieving service health.
+func Health(service string) http.HandlerFunc {
+	return http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
+		w.Header().Add(contentType, contentTypeJSON)
+		if r.Method != http.MethodGet && r.Method != http.MethodHead {
+			w.WriteHeader(http.StatusMethodNotAllowed)
+			return
+		}
+
+		res := HealthInfo{
+			Status:      svcStatus,
+			Description: service + " service",
+			Version:     version,
```

Comment:
it would be good to have timestamp of build so that when debugging deployment by querying the health you can confirm that right version is deployed

Reply:
It's what @blokovi proposed. There an issue about: https://github.com/mainflux/mainflux/issues/1543

Reply:
@mteodor there is a way to put timestamp and Git hash during the compilation phase. We had this before. I think it would make sense to add it back.